.github/workflows/build-pr.yml

```diff
@@ -32,7 +32,16 @@ jobs:
         echo "::remove-matcher owner=eslint-compact::"
         echo "::remove-matcher owner=eslint-stylish::"
     - run: bun run all
-    - run: git status
-    - uses: stefanzweifel/git-auto-commit-action@v7
+    - run: |
+        DIFF=$(git diff data/)
+        echo "DIFF<<EOF" >> $GITHUB_ENV
+        echo "$DIFF" >> $GITHUB_ENV
+        echo "EOF" >> $GITHUB_ENV
```

Comment:
I don't really understand this, but it's what GH needs for multiline env variables: https://docs.github.com/en/actions/reference/workflows-and-actions/workflow-commands#multiline-strings